### ISSUE TITLE
[SPIKE] Compare WA value sets to nationally-accepted ones (HL7, FHIR, etc.)

### ISSUE THREAD

**Issue by jsutantio** [CDCgov/prime-reportstream#11590]
### User Story
As a ReportStream architect, I want to understand Washington state's ELR requirements (specifically around their custom value sets), so that I can appropriately define validation rules and requirements that accommodate all STLTs (including WA).

### Description/Use Case
In an effort to create baseline non-receiver-specific validation rules, we seek to understand and enforce the common requirements across all STLTs. Many STLTs reference the HL7 v2.5.1 international Implementation Guide; however, WA has unique rules when it comes to reporting race and ethnicity, which may greatly differ from other STLTs. The goal is to identify how different WA's rules and code systems are - or if there exists a standard (widely-used) HL7 or FHIR code system that may pass WA's strict rules.

### Question to answer
- Are the common value sets (provided by HL7, FHIR, or any other (inter)national orgs) comprehensive enough that they encapsulate all the values listed in WA's specific value sets? 
- If not, which values are not included?
- If so, which value set/ code system do you recommend ReportStream references? A combination of tables?

### Notes
Although we've identified race (PID.10) and ethnicity (PID.22) as dig sites, there may be other data types that WA has specific value sets that may differ from what's commonly used nationally. As part of the acceptance criteria, identify any other data types that are within the [Prioritized HL7 Fields](https://docs.google.com/spreadsheets/d/1MBR0-eO0pUzmVZVQzPJ6MMV4SyFz-Tf7mUdkwOn-heY/edit#gid=826891331) (excluding MSH, NTE, and SFT segments) that will also benefit from comparing between WA-specific tables and those used nationally. You'll need to look at WA's implementation guide or website to identify what other value sets they use for these other fields (if applicable).

**Helpful Resources (but probably not comprehensive):**
[HL7 v2.5.1 Implementation Guide](https://dhhs.ne.gov/epi%20docs/HL7-2.5.1-Guide.pdf)
[HL7 0005- Race table](https://hl7-definition.caristix.com/v2/HL7v2.5.1/Tables/0005)
[HL7 CodeSystem: Race](https://terminology.hl7.org/5.2.0/CodeSystem-v3-Race.html)
[HL7 0189-Ethnic Group table](https://hl7-definition.caristix.com/v2/HL7v2.5.1/Tables/0189)
[FHIR ValueSet: OMB Race Categories](http://hl7.org/fhir/us/core/ValueSet/omb-race-category)
[FHIR ValueSet: Detailed Race](http://hl7.org/fhir/us/core/ValueSet/detailed-race)
[CDC Race and Ethnicity Code Set](https://www.cdc.gov/nchs/data/dvs/race_ethnicity_codeset.pdf)
[WA-specific ethnicity, race, and preferred language information](https://app.leg.wa.gov/WAC/default.aspx?cite=246-101-011)

### Acceptance Criteria
- [x] Identify any other data types that are within the [Prioritized HL7 Fields](https://docs.google.com/spreadsheets/d/1MBR0-eO0pUzmVZVQzPJ6MMV4SyFz-Tf7mUdkwOn-heY/edit#gid=826891331) (excluding MSH, NTE, and SFT segments) that will also benefit from comparing between WA-specific tables and those used nationally
- [x] Comparison of WA-specific value sets/ codes against value sets and code systems of the equivalent data type (i.e., race, ethnicity, and any other pertinent data elements) for HL7 and FHIR.
- [x] Documentation of similarities and differences between WA's values sets and those defined by HL7 and FHIR specs (can be made as an additional sheet in the [Receiver HL7 Segment Requirements Analysis](https://docs.google.com/spreadsheets/d/1MBR0-eO0pUzmVZVQzPJ6MMV4SyFz-Tf7mUdkwOn-heY/edit?usp=sharing) spreadsheet)
- [x] Answer to the "questions to answer" (document in this ticket and discussed with product owner)

**Comment by dylansmith-Nava** [comment]
CC @jillian-hammer for your eyes to check out!

**Comment by dylansmith-Nava** [comment]
@jsutantio I wrote up a [synthesis document](https://docs.google.com/document/d/1SNnPHYQa0jtZt38g78nDMYVGUX8bama7v1dI0UA4Dvw/edit) based on some of the research and conversations I've been having about WA value sets. Most of these answers appear to be in the 'WA' column within the HL7 field comparison sub-tab. I checked the first box under Acceptance Criteria because that seems to be done. Hopefully these notes help!
